frontend/src/components/CheckboxWithIcon/index.js

```diff
@@ -0,0 +1,41 @@
+import { PropTypes } from "prop-types";
+
+import styles from "./index.module.scss";
+
+function CheckboxWithIcon({ id, icon, label, checked, disabled, onChange }) {
+  return (
+    <div className={styles.root}>
+      <input
+        id={id}
+        className={styles.input}
+        type="checkbox"
+        disabled={disabled}
+        checked={checked}
+        onChange={onChange}
+      />
+
+      <div className={styles.control}>
+        <img alt="icon" src={icon} className={styles.icon}></img>
```

Comment:
Let's use `ReactComponent` > https://create-react-app.dev/docs/adding-images-fonts-and-files/#adding-svgs